release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3021,6 +3021,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+// 4.18.14
+[id="ocp-4-18-14_{context}"]
+=== RHSA-2025:7863 - {product-title} {product-version}.14 bug fix update and security update
+
+Issued: 20 May 2025
+
+{product-title} release {product-version}.14 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7863[RHSA-2025:7863] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7865[RHBA-2025:7865] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.14 --pullspecs
+----
+
+[id="ocp-4-18-14-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the omission of the `OLMManagedLabelKey` label on objects resulted in cluster operation failures. With this release,  an update improves pod stability and ensures that the Operator Lifecycle Manager operates properly. (link:https://issues.redhat.com/browse/OCPBUGS-56098[OCPBUGS-56098])
+
+* Previously, an invalid tar extraction format resulted in an improper file separation in the `ramdisk` logs, and caused file separators to appear randomly. With this release, an updated `ramdisk` log file  processes tar entries individually. This fix improves log readability, making them easier to interpret. (link:https://issues.redhat.com/browse/OCPBUGS-55938[OCPBUGS-55938])
+
+* Previously, incorrectly formatted proxy variables in an external binary resulted in build failures. With this release, an update removes proxy environment variables from the build pod and prevents any build failures. (link:https://issues.redhat.com/browse/OCPBUGS-55699[OCPBUGS-55699])
+
+* Previously, there was no event logged when an error occurred from failed conversion from ingress to route conversion. With this update, this error appear in the event logs. (link:https://issues.redhat.com/browse/OCPBUGS-55338[*OCPBUGS-55338*])
+
+* Previously, a missing `afterburn` package resulted in the failure of the `gcp-hostname.service`, which causied the `scale-up` job to fail, impacting end user deployments. With this release, the `afterburn` package is installed in the RHEL `scale-up` job. This fix enables a successful `scale-up` action, resolving the `gcp-hostname` service failure. (link:https://issues.redhat.com/browse/OCPBUGS-55158[*OCPBUGS-55158*])
+
+* Previously, there was no communication between a `localnet` pod and a pod in the default network when both pods are on the same node. With this release, an update fixes this problem when the endpoints are on a different node. (link:https://issues.redhat.com/browse/OCPBUGS-55016[*OCPBUGS-55016*])
```

Comment:
```suggestion
* Previously, there was no communication between a `localnet` pod and a pod in the default network when both pods were on the same node. With this release, an update fixes this problem when the endpoints are on a different node. (link:https://issues.redhat.com/browse/OCPBUGS-55016[*OCPBUGS-55016*])
```
I'm a little confused by this bug-fix description. The first sentence sounds like the problem happens when both pods are on the same node. The second sentence sounds like the problem is fixed when endpoints are on different nodes. Is that correct?

Reply:
Thanks @lahinson, I updated the bug text and fixed the wording.